### ISSUE TITLE
Undocumented installation step for macOS

### ISSUE THREAD

**Issue by AustinEast** [jcward/hxtelemetry#65]
Hey, I recently re-installed HxScout on a new Mac, and I ran into an issue with getting the app started. Every time I would open it, it would crash after a couple of seconds. I had no idea what I did wrong, wasting a bunch of time troubleshooting until I remembered that last time I needed to move the App to my `Applications` directory for it to start up without crashing.

I did this and everything is now working fine :)

Long story short, I think this step for installing on OSX should be documented somewhere, just so we can save devs some time from scratching their heads in confusion.
